server/src/main/java/org/elasticsearch/cluster/service/ClusterApplierService.java

```diff
@@ -134,7 +134,7 @@ protected synchronized void doStart() {
         addListener(localNodeMasterListeners);
         threadPoolExecutor = EsExecutors.newSinglePrioritizing(
                 nodeName + "/" + CLUSTER_UPDATE_THREAD_NAME,
-                daemonThreadFactory(settings, CLUSTER_UPDATE_THREAD_NAME),
+                daemonThreadFactory(nodeName, CLUSTER_UPDATE_THREAD_NAME),
```

Comment:
I added this method to `EsExecutors` because I like how explicit it is. We already have the node name in a few places and *don't* have settings in this one.